src/main/java/streams/cta/io/CSVWriter.java

```diff
@@ -0,0 +1,74 @@
+package streams.cta.io;
+
+import com.google.common.base.Joiner;
+import stream.Data;
+import stream.Keys;
+import stream.ProcessContext;
+import stream.StatefulProcessor;
+import stream.annotations.Parameter;
+import stream.io.SourceURL;
+
+import java.io.FileOutputStream;
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * CSVWriter writes out the values for the given keys to a csv file.
+ * The first line of the CSV file is a header.
+ * The header contains the names of the keys found in the data item.
+ *
+ * @author kai
+ */
+public class CSVWriter implements StatefulProcessor {
```

Comment:
Otherwise it might be more helpful to write different CTA Events in parallel in different files!?